lib/versioning/VersionID.js

```diff
@@ -202,8 +211,19 @@ function base62Decode(str) {
         const enc2 = str.substring(start, start + B62V_EPAD.length);
         const orig2 = base62Integer.decode(enc2);
         start += B62V_EPAD.length;
-        const enc3 = str.substring(start);
+        let enc3 = str.substring(start);
+        // take off prefix 0s which represent null bytes
+        let idx = 0;
+        while (idx < enc3.length) {
+            if (enc3[idx] === '0') {
+                idx++;
+            } else {
+                break;
+            }
+        }
+        enc3 = enc3.slice(idx);
```

Comment:
In JS this should be relatively efficient and simplifies code a bit:

```suggestion
        enc3 = enc3.replace(/^0*/, '');
```

Reply:
leaving as is for a bit of additional perf: https://measurethat.net/Benchmarks/Show/18405/0/regex-vs-slice